CDP4CrossViewEditor/Views/UserControls/ElementThingSelector.xaml.cs

```diff
@@ -25,6 +25,10 @@
 
 namespace CDP4CrossViewEditor.Views.UserControls
 {
+    using CDP4CrossViewEditor.ViewModels;
```

Comment:
Do we need to add new using clauses if there's no new code in this class?

Reply:
No.